amd_openvx_extensions/amd_rpp/include/internal_rpp.h

```diff
@@ -74,14 +75,25 @@ enum vxTensorLayout {
     VX_NTF = 6      // Time major, Used for Spectrogram/MelFilterBank
 };
 
+const std::map<vxTensorLayout, RpptLayout> TENSOR_LAYOUT_MAPPING = {
```

Comment:
Do we need this in all capitals? It is just a map variable name.

Reply:
Renamed the variable to tensorLayoutMapping